packages/mds-transaction-service/service/stream.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import stream, { StreamProducer } from '@mds-core/mds-stream'
+import { getEnvVar } from '@mds-core/mds-utils'
+import { TransactionDomainModel } from '../@types'
+
+const { TENANT_ID } = getEnvVar({
+  TENANT_ID: 'mds'
+})
+
+const transactionEventTopic = [TENANT_ID, 'transaction'].join('.')
+
+export type KafkaTransaction = TransactionDomainModel // TODO is this needed?
```

Comment:
I think you could just use `TransactionDomainModel` directly without the type alias

Reply:
I think the type alias is helpful so if we ever need to change the kafka type we can get compiler errors that show us exactly what needs changing, versus having to search for TransactionDomainModel and figure out what needs changing.

Reply:
I'm not sure what situation you're thinking of where this alias would prove more helpful than just using the type directly... Would you mind putting together an example? The compiler ought to give pretty clear pointers to the exact line that's failing in almost every case.

Reply:
Suppose we decide that we don't want to record the numeric row ID in kafka after all. Then we'd want to redefine the KafkaTransaction type to not have that field. Then we'd see errors in the parts of the code base where we would pass that field in, and modify them.